src/python/k4a/src/k4a/_bindings/k4a.py

```diff
@@ -0,0 +1,525 @@
+k4a_device_get_color_control.restype = EStatus
+k4a_device_get_color_control.argtypes = (_DeviceHandle, _ctypes.c_int, _ctypes.POINTER(_ctypes.c_int), _ctypes.POINTER(_ctypes.c_int32))
+
+
+#K4A_EXPORT k4a_status_t k4a_device_set_color_control(k4a_device_t device_handle,
+#                                                     k4a_color_control_command_t command,
+#                                                     k4a_color_control_mode_t mode,
+#                                                     int32_t value);
+k4a_device_set_color_control = _k4a_lib.k4a_device_set_color_control
+k4a_device_set_color_control.restype = EStatus
+k4a_device_set_color_control.argtypes = (_DeviceHandle, _ctypes.c_int, _ctypes.c_int, _ctypes.c_int32)
+
+
+#K4A_EXPORT k4a_buffer_result_t k4a_device_get_raw_calibration(k4a_device_t device_handle,
+#                                                              uint8_t *data,
+#                                                              size_t *data_size);
+k4a_device_get_raw_calibration = _k4a_lib.k4a_device_get_raw_calibration
+k4a_device_get_raw_calibration.restype = EBufferStatus
+k4a_device_get_raw_calibration.argtypes = (_DeviceHandle, _ctypes.POINTER(_ctypes.c_uint8), _ctypes.POINTER(_ctypes.c_ulonglong))
+
+
+#K4A_EXPORT k4a_status_t k4a_device_get_calibration(k4a_device_t device_handle,
+#                                                   const k4a_depth_mode_t depth_mode,
+#                                                   const k4a_color_resolution_t color_resolution,
+#                                                   k4a_calibration_t *calibration);
+k4a_device_get_calibration = _k4a_lib.k4a_device_get_calibration
+k4a_device_get_calibration.restype = EStatus
+k4a_device_get_calibration.argtypes = (_DeviceHandle, _ctypes.c_int, _ctypes.c_int, _ctypes.POINTER(_Calibration))
+
+
+#K4A_EXPORT k4a_status_t k4a_device_get_sync_jack(k4a_device_t device_handle,
+#                                                 bool *sync_in_jack_connected,
+#                                                 bool *sync_out_jack_connected);
+k4a_device_get_sync_jack = _k4a_lib.k4a_device_get_sync_jack
+k4a_device_get_sync_jack.restype = EStatus
+k4a_device_get_sync_jack.argtypes = (_DeviceHandle, _ctypes.POINTER(_ctypes.c_bool), _ctypes.POINTER(_ctypes.c_bool))
+
+
+#K4A_EXPORT k4a_status_t k4a_calibration_get_from_raw(char *raw_calibration,
+#                                                     size_t raw_calibration_size,
+#                                                     const k4a_depth_mode_t depth_mode,
+#                                                     const k4a_color_resolution_t color_resolution,
+#                                                     k4a_calibration_t *calibration);
+k4a_calibration_get_from_raw = _k4a_lib.k4a_calibration_get_from_raw
+k4a_calibration_get_from_raw.restype = EStatus
+k4a_calibration_get_from_raw.argtypes = (_ctypes.POINTER(_ctypes.c_char), 
```

Comment:
there is a ctpyes c_size_t
That might be more extensible if we want to support both 32 and 64 bit

Reply:
Changed to c_size_t.